packages/react-dom/README.md

```diff
@@ -13,38 +13,48 @@ npm install react react-dom
 ### In the browser
 
 ```js
-var React = require('react');
-var ReactDOM = require('react-dom');
+import { createRoot } from 'react-dom';
```

Comment:
@gaearon 
Should it not be imported from `'react-dom/client'` instead? Or does it not make a difference?
https://reactjs.org/blog/2022/03/29/react-v18.html#react-dom-client

Reply:
oh yeah.

Reply:
(there's one more in `packages/react/README.md`)